datasets/fr/tresor/fr_tresor_gels_avoir.yml

```
@@ -5043,3 +5043,52 @@ lookups:
       - match: "Numéro d'enregistrement : 1022601930537"
         props:
           registrationNumber: 1022601930537
+      - match: "Type d’entité: société par actions - Principal établissement: Fédération de Russie: /"
+        props:
+          jurisdiction: ru
+          legalForm: société par actions
+          address: "Fédération de Russie"
+      - match: "Numéro d’enregistrement: 1023202741781"
+        props:
+          ogrnCode: 1023202741781
+      - match: "Type d’entité: société à responsabilité limitée - Principal établissement: Fédération de Russie: /"
+        props:
+          jurisdiction: ru
+          legalForm: société à responsabilité limitée
+          address: "Fédération de Russie"
+      - match: "Numéro d’identification fiscale: 1646043699"
+        props:
+          innCode: 1646043699
+      - match: "Type d’entité: société à responsabilité limitée - Principal établissement: République populaire de Chine/Hong Kong: /"
+        props:
+          jurisdiction: hk
+          legalForm: société à responsabilité limitée
+          address: "République populaire de Chine/Hong Kong"
+      - match: "Numéros d’enregistrement: 2182045 64199990-000 CR No. 2182045"
+        props:
+          registrationNumber: 2182045 64199990-000 CR No. 2182045
```

Looks like these are distinct numbers that we'd benefit from including as distinct values
https://sanctionssearch.ofac.treas.gov/Details.aspx?id=43272

```suggestion
          registrationNumber:
            - 2182045
            - 64199990-000
```